### PR TITLE
Add kiali and tracing component to Kyma Alpha deploy flow

### PR DESCRIPTION
**Pull request by JeremyHarisch**
**Description**

Changes proposed in this pull request:

- Added `kiali` and `tracing` component to Kyma alpha deploy flow by  adding them to `components.yaml`
- kcproxy should be deactivated for both components in alpha deploy flow, since those two services are crashing while starting up (CrashloopBackOff) --> Please review this [PR](https://github.com/kyma-project/hydroform/pull/268) too

**Related issue(s)**

Fixes [#850](https://github.com/kyma-project/cli/issues/850)

### CODE REVIEW

**Review by clebs — DISMISSED** [review]

File: resources/kiali/values.yaml [hunk 1]
```diff
@@ -27,7 +27,7 @@ global:
     enabled: true
 
 kcproxy:
-  enabled: true
+  enabled: false
```
Comment (skhalash) [inline resources/kiali/values.yaml:30]: What is the impact of this change on SKR?

**Comment by JeremyHarisch** [comment]
Issue will be fixxed with other solution: [PR]()

**Review by skhalash — DISMISSED** [review]

**Review by clebs — APPROVED** [review]

**Review by colunira — APPROVED** [review]
/meow

**Pull request merged**